test/runner.js

```diff
@@ -0,0 +1,23 @@
+
+var isNode = typeof window === 'undefined'
+
+if (isNode) {
+  before(function(next) {
+    require('jsdom').env({
```

Comment:
I'd really want to use simple-dom globally in Riot. It's already being used for server side generation on this [pull request](https://github.com/muut/riotjs/pull/368)